test/run_tests.sh

```diff
@@ -105,8 +114,10 @@ function run_pjrt {
 }
 
 function run_async_scalar {
-  echo "Running in Async Scalar Upload mode: $@"
-  XLA_TRANSFER_SCALAR_ASYNC=1 run_test "$@"
+  if [ -Z "$PJRT_DEVICE" ] ; then
```

Comment:
check `PJRT_DEVICE` in run_async_scalar func because many tests failed with almost the same error message:
```
Traceback (most recent call last):
     File "/pytorch/xla/test/test_operations.py", line 703, in test_random_from_to_bool
     self.assertTrue(from_val <= x.to(torch.int).min() < (from_val + delta))
 RuntimeError: ./tensorflow/compiler/xla/xla_client/pjrt_computation_client.h:106 : ExecuteChained not implemented
```

https://pantheon.corp.google.com/cloud-build/builds/f3d7c450-5147-405e-b1a8-e53f86e96ecb?project=tpu-pytorch&jsmode=o&mods=allow_workbench_image_override